api/src/main/java/com/velocitypowered/api/event/connection/PreLoginEvent.java

```diff
@@ -75,6 +75,7 @@ public String getUsername() {
    * Returns the UUID of the connecting player. This value is {@code null} on 1.19.1 and lower.
    *
    * @return the uuid
+   * @sinceMinecraft 1.19.1
```

Comment:
Is this right? Because if it is null in 1.19.1 and below, why was it added in 1.19.1?

Reply:
fixed